### DISCUSSION TITLE
:= vs = as latex output

### DISCUSSION THREAD

**Discussion by webdeb** [josdejong/mathjs#2987] [Ideas]
This is not a bug, more a discussion. The latex output of any assignment produces `:=` (Which is a definition in math, and it can be done this way)

However, if you look on Wikipedia, or most other resources, the simple `=` is used in most of the formulas.
This can lead to confusion for most of the users, because they are not used to it. On the other hand, professionals which know both forms would not be confused anyway..

I would like to propose to replace `:=` by `=` for the `toTex()` output. Because it's also not produced by `toHTML` or `toString` methods.

And in a further step, let the user decide, by using either `=` or `:=` in their math string, so the output is more explicit and predictable.

**Comment by josdejong** [comment]
Thanks for your suggestion. I guess we can do that, I'm not sure what is most widely used though. Are there any side effects that you can think of? Are there any systems that put a meaning in either `:=` or `=`?

It will be a breaking change, so we need to mark the change as such.

**Reply by josdejong** [reply]
For reference, see #2980

**Reply by josdejong** [reply]
The assignment operator used in `.toTex()` is now changed from `=` to `:=` in mathjs `v12.0.0`
